build.gradle

```diff
@@ -154,23 +154,72 @@ modrinth {
     String projectVersion = project.version.toString()
     List<String> projectVersionNumber = projectVersion.split("-", 2)
     String projectVersionName = "Release ${projectVersionNumber[0]}"
-    if (projectVersion.contains("beta")) {
-        List<String> projectBeta = projectVersionNumber[1].split("[.]", 2)
+    if (projectVersion.contains("patch")) {
+        List<String> _projectVersionNumber = projectVersion.split("\\+", 2)
+        List<String> projectPatch = _projectVersionNumber[1].split("\\.", 2)
```

Comment:
since this is used in every branch, could it be defined once (before the if statement)

Reply:
Modified it to do so and likely improve the system a bit